site/_release/0.2.0.md

```diff
@@ -252,6 +252,6 @@ $ git shortlog -sn apache-arrow-0.1.0..apache-arrow-0.2.0
 * [ARROW-561](https://issues.apache.org/jira/browse/ARROW-561) - Update java & python dependencies to improve downstream packaging experience
 
 [2]: https://github.com/apache/arrow/releases/tag/apache-arrow-0.2.0
-[3]: https://dist.apache.org/repos/dist/release/arrow/arrow-0.2.0/apache-arrow-0.2.0.tar.gz.md5
-[6]: https://dist.apache.org/repos/dist/release/arrow/arrow-0.2.0/apache-arrow-0.2.0.tar.gz
-[7]: https://dist.apache.org/repos/dist/release/arrow/arrow-0.2.0/apache-arrow-0.2.0.tar.gz.asc
+[3]: https://www-us.apache.org/dist/arrow/arrow-0.2.0/apache-arrow-0.2.0.tar.gz.md5
```

Comment:
Instead of a mirror, this probably should be in the form of `https://www.apache.org/dyn/closer.cgi/parquet/apache-parquet-cpp-1.1.0/`